boards/common/remote/include/periph_common.h

```diff
@@ -78,17 +78,26 @@ static const timer_conf_t timer_config[] = {
  * @{
  */
 static const uart_conf_t uart_config[] = {
+    /* UART0 is mapped to debug usb */
     {
         .dev      = UART0_BASEADDR,
-        .rx_pin   = GPIO_PIN(0, 0),
-        .tx_pin   = GPIO_PIN(0, 1),
+        .rx_pin   = GPIO_PIN(PORT_A, 0),
```

Comment:
In the past I learned that people get confused when they look at a `periph_conf` file and (CPU specific) expressions such as `PORT_A` are used which won't work in a generic way from the application code. Obviously, using literals is nicer in general. What's your opinion on that?

Reply:
> [...] Obviously, using literals is nicer in general. [...]

I strongly object to this proposition.